java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture1/tasks/_4/Number4.java

```diff
@@ -0,0 +1,62 @@
+package com.zoolatech.lecture1.tasks._4;
+
+import java.util.Scanner;
+
+/**Write a program that accepts a day of the week as an integer number
+ *  (1 - Monday, 2 - Tuesday, …, 7 - Sunday) and output “Need to go to work…”
+ *  if the day is a weekday or “Sleeping…” if it’s a weekend.
+ *  Create two versions of the program:
+ *  one should use the if-else statement, second - the conditional operator.
+ */
+
+public class Number4 {
+    public static void main(String[] args) {
+        Number4 number4 = new Number4();
+        number4.ifElse();
+        System.out.println("If-Else");
+        number4.conditionalOperator();
+        System.out.println("Conditional Operator");
+    }
+
+    //MARK: FIRST VERSION
+    private void ifElse() {
+        int dayOfAWeek;
+        RangeOfDays range = new RangeOfDays(1, 5);
```

Comment:
Try to find a better naming for the `range` variable: as for now, it's not clear how it can be used and what data it contains